### DISCUSSION TITLE
"Add to favorite"

### DISCUSSION THREAD

**Discussion by AXEN-hub** [Droid-ify/client#111] [Ideas]
Hello! I kindly love this app because it's fast and clean, for that reason i decided to switch to it. But i'm missing so much the feature of "Add to favorite". Can you add this feature into the app? Please. Thanks in advice!

**Comment by Iamlooker** [comment]
Ya, I will be working on this soon, as soon as I am done refactoring the data layer of the app :)

**Comment by Iamlooker** [comment]
Done
